bot.py

```diff
@@ -0,0 +1,77 @@
+import heuristics
+import board_heu
+from game import Game, Player, BOARD_SIZE
+from draughts import Board, Move
+import time
+
+"""
+NEW alpha-beta pruning
+
+TO DO:
+- Clean up code
+- For now, everything minimax-related is played on the Board (not Game) 
+  --> change that
+- Problem with moves that are evaluated the same: the model will have a tendency to play a little too much on the left (or right) 
+  --> implement some randomness
+- Minimax takes too long to compute 
+  --> alpha-beta pruning
+- Update alpha-beta pruning in all the files
```

Comment:
This is implemented. Let's keep one version of the minmax algorithm with ab-pruning so we don't need to update it everywhere